content/news/XXXX-XX-XX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,107 @@
+Once you've created an Effect Chain, you can create a custom Super Knob mapping which turns the chain into a Quick-Effect, where you take creative control with a single knob.
+Effect parameters can be reordered and or hidden.
+In the effect units, Mixxx 2.4 also displays the value and unit of effect knob parameters while they are being adjusted.
+
+### Track Menu Enhancements
+
+The track menu in Mixxx 2.4 is enhanced by several new features:
+
+- Select loaded track in library.
+- Analyze/re-analyze with variable/constant BPM on a per-track basis.
+- Look up track metadata at Discogs, SoundCloud and LastFM.
+- Remove track files from disk.
+
+### Sync Lock Improvements
+
+Mixxx 2.4 adds support for manually selecting the Sync Leader by clicking on the new button with the crown symbol next to the Sync button.
+This mode is useful when tracks have variable BPM.
+All decks with sync enabled will follow the changing BPM of the Sync Leader.
+@Video(https://www.youtube.com/watch?v=REiG8DBKr-M)
+
+### Many Other Changes
+
+#### Soundstretcher / Key-Shift Algorithms
+
+Mixxx 2.4 now supports three different key shift algorithms:
+
+- Soundtouch (Basic audio quality - medium CPU usage)
+- Rubberband R2 (Good audio quality - higher CPU usage)
+- Rubberband R3 (High audio quality - very high CPU usage)
+
+#### Waveform Rendering Reworked
+
+The waveform display has been rewritten, solving some longstanding performance issues, especially on macOS.
+
+#### Apple Silicon Support
+
+Mixxx now supports macOS ARM systems ("Apple Silicon") such as M1 natively and provides installation packages for those architectures.
+For macOS users, this increases the stability and performance of Mixxx on Apple silicon.
+
+#### Controller Backend
+
+Modernized Javascript ES7 based controller mapping system.
+HID backend rework.
+
+### Mixxx 2.4 Factsheet
+
+#### Supported Platforms
+
+- Windows 7 or later on 64bit processors
+- macOS 10.12 or later on Intel processors
+- macOS 11.0 or later on ARM processors (Apple Silicon)
+- Ubuntu Linux 20.04 or later (other Linux distributions are supported by third parties)
+
+#### Supported Controllers
+
+- Mappings for 129 MIDI controllers included
+- Mappings for 16 HID controllers included
+
+See the [full list](https://manual.mixxx.org/2.4/hardware/manuals#controllers) of supported controllers in the Mixxx Manual.
+
+### Press Information
+
+Mixxx 2.4 is available right now:
+[https://mixxx.org/download/](https://mixxx.org/download/)
+
+The manual for Mixxx 2.4 can be found here:
+[https://manual.mixxx.org/2.4/en/](https://manual.mixxx.org/2.4/en/)
+
+#### Images For Media Use
+
+High-Resolution images of the Mixxx logo can be downloaded here:
+[https://mixxx.org/press/logo](https://mixxx.org/press/logo)
+
+Full-Resolution images of the Mixxx 2.4 screenshots above can be downloaded here:
+[https://mixxx.org/screenshots](https://mixxx.org/screenshots)
```

Comment:
We need to merge the screenshot PR first

Reply:
order does not matter because these are all going to the 2.4 branch, which will be merged as one piece